### ISSUE TITLE
Loading Chartist dependecy fails under npm@3

### ISSUE THREAD

**Issue by dsibilly** [panosoft/node-chartist#6]
The following code fails to run when node-chartist is installed via npm@3:

```javascript
const generate = require('node-chartist');
```

The error message is as follows:
```
fs.js:640
  return binding.open(pathModule._makeLong(path), stringToFlags(flags), mode);
                 ^

Error: ENOENT: no such file or directory, open '/Users/dsibilly/Documents/Projects/chartist-test/node_modules/node-chartist/node_modules/chartist/dist/chartist.js'
    at Error (native)
    at Object.fs.openSync (fs.js:640:18)
    at Object.fs.readFileSync (fs.js:508:33)
    at Object.<anonymous> (/Users/dsibilly/Documents/Projects/chartist-test/node_modules/node-chartist/lib/chart.js:14:26)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
```

I believe this to be due to the readFileSync call on in [lib/chart.js:14](https://github.com/panosoft/node-chartist/blob/master/lib/chart.js#L14):
```javascript
var chartistSource =  fs.readFileSync(path.resolve(__dirname, '../node_modules/chartist/dist/chartist.js'));
```

The line assumes that depdencies will be installed in a nested structure as in npm@2 and earlier. Given that npm@3 [flattens dependencies](https://docs.npmjs.com/how-npm-works/npm3) to eliminate some of the nesting hell that could result, this assumption will not work with the latest tools.

**Comment by dsibilly** [comment]
I have addressed this issue and submitted #7 for consideration.

**Comment by alexgig** [comment]
Thanks. You're pull request has been merged.

Best,
Alex
